src/test/java/org/jabref/SearchQueryHighlightListenerTest.java

```diff
@@ -34,6 +26,7 @@ public void setUp() {
         when(loader.getProtectedTermsLists()).thenReturn(Collections.emptyList());
     }
 
+    /*
```

Comment:
The problem is these tests don't compile anymore (TextArea no longer gives access to the highlighter). I marked it now as a todo for later.